src/main/java/io/stargate/sgv2/jsonapi/service/operation/collections/FindCollectionOperation.java

```diff
@@ -49,13 +48,11 @@ public record FindCollectionOperation(
     int maxSortReadLimit,
     boolean singleResponse,
     float[] vector,
+    SortExpression bm25SearchExpression,
```

Comment:
Like existing entry for vector ANN value, not ideal, but follows existing pattern.